src/control/server/mgmt_svc.go

```diff
@@ -312,10 +317,39 @@ func checkIsMSReplica(mi *IOServerInstance) error {
 	return nil
 }
 
+// validatePool ensures minimum SCM/NVMe pool size per storage target.
+func (svc *mgmtSvc) validatePool(req *mgmtpb.PoolCreateReq) error {
```

Comment:
nit: Naming this `validatePoolCreate()` would be a little clearer. Although if you switch the min values to numeric consts you could probably just collapse the simplified logic back into `PoolCreate()`. Breaking it out does make it a tighter unit test, but IMO that comes at the expense of test coverage in `PoolCreate()` which as far as I can see has no test coverage at all right now.